### PR TITLE
enabled Hound

### PR DESCRIPTION
**Pull request by geoff-reid**
- added .hound.yml, .flake8.ini, and mkcheck.sh files
 - cleaned up flake8 errors

@RackHD/corecommitters, @tannoa2 @keedya @uppalk1

### CODE REVIEW

File: controllers/default_controller.py [hunk 1]
```diff
@@ -21,22 +21,23 @@ def systemGetAll(host = None, user = None, password = None):
             units = []
             components = handle.query_children(in_dn="sys", class_id=x["ciscoXmlName"])
```
Comment (rahmanmuhamad) [inline controllers/default_controller.py]: Do you need to check components for null or size before iterating over it ?

Comment (rahmanmuhamad) [inline controllers/default_controller.py]: Do you need to check components for null or size before iterating over it ?

Comment (rahmanmuhamad) [inline controllers/default_controller.py]: Do you need to check components for null or size before iterating over it ?

File: controllers/default_controller.py [hunk 2]
```diff
@@ -21,22 +21,23 @@ def systemGetAll(host = None, user = None, password = None):
...
         return finalObjs
 
-def getRackmount(host = None, user = None, password = None):
+
+def getRackmount(host=None, user=None, password=None):
     data = []
     handle = UcsHandle(host, user, password, secure=False)
     if handle.login():
         computeRackUnit = handle.query_children(in_dn="sys", class_id="computeRackUnit")
         for x in computeRackUnit:
```
Comment (rahmanmuhamad) [inline controllers/default_controller.py]: do you want to check computeRackUnitr null ?

Comment (rahmanmuhamad) [inline controllers/default_controller.py]: do you want to check computeRackUnitr for null ?

File: controllers/default_controller.py (hunk 1 above)
Comment (geoff-reid) [inline controllers/default_controller.py]: This PR is only for the enabling of hound.  This issue will be addressed as part of [RAC-4150](https://rackhd.atlassian.net/browse/RAC-4150)

File: controllers/default_controller.py (hunk 2 above)
Comment (geoff-reid) [inline controllers/default_controller.py]: This PR is only for the enabling of hound.  This issue will be addressed as part of [RAC-4150](https://rackhd.atlassian.net/browse/RAC-4150)

Comment (geoff-reid) [inline controllers/default_controller.py]: This PR is only for the enabling of hound.  This issue will be addressed as part of [RAC-4150](https://rackhd.atlassian.net/browse/RAC-4150)

**Pull request merged**